tests/perf/test_accuracy.py

```diff
@@ -2,142 +2,255 @@
+            model = ov.Core().read_model(model_path)
+            return model, None
+
+        elif model_name in IMAGENET_MODELS + TRANSFORMER_MODELS:
+            self.dataset_label_list = IMAGENET_LABELS
+            _, model_cfg = convert_timm_to_ir(model_name, data_dir, self.supported_num_classes)
+            ir_path = data_dir / "timm_models" / "converted_models" / model_name / "model_fp32.xml"
+            model = ov.Core().read_model(ir_path)
+            return model, model_cfg
+        else:
+            raise ValueError(f"Model {model_name} is not supported since it's not VOC or ImageNet model.")
+
+    def setup_process_fn(self, model_cfg):
+        if self.model_name in VOC_MODELS:
+            # VOC model
+            self.preprocess_fn = get_preprocess_fn(
+                change_channel_order=False,
+                input_size=(224, 224),
+                hwc_to_chw=True,
+            )
+            self.postprocess_fn = get_postprocess_fn(activation=ActivationType.SIGMOID)
+        elif self.model_name in IMAGENET_MODELS + TRANSFORMER_MODELS:
+            # Timm ImageNet model
+            mean_values = [(item * 255) for item in model_cfg["mean"]]
+            scale_values = [(item * 255) for item in model_cfg["std"]]
+            self.preprocess_fn = get_preprocess_fn(
+                change_channel_order=True,
+                input_size=model_cfg["input_size"][1:],
+                mean=mean_values,
+                std=scale_values,
+                hwc_to_chw=True,
+            )
+            self.postprocess_fn = get_postprocess_fn(activation=ActivationType.SOFTMAX)
+        else:
+            raise ValueError(f"Model {self.model_name} is not supported since it's not VOC or ImageNet model.")
+
+    def setup_explainer(self, model, explain_method):
+        explain_mode = ExplainMode.WHITEBOX if explain_method in WhiteBoxXAIMethods else ExplainMode.BLACKBOX
+
+        if self.model_name in TRANSFORMER_MODELS and explain_method == Method.RECIPROCAM:
+            explain_method = Method.VITRECIPROCAM
 
         self.explainer = Explainer(
             model=model,
             task=Task.CLASSIFICATION,
             preprocess_fn=self.preprocess_fn,
-            explain_mode=ExplainMode.WHITEBOX,
+            postprocess_fn=self.postprocess_fn,
+            explain_mode=explain_mode,
+            explain_method=explain_method,
+            embed_scaling=True,
         )
+        kwargs = {}
+        if explain_method in BlackBoxXAIMethods:
+            # TODO: Make Preset configurable as well
+            kwargs.update({"preset": Preset.SPEED})
+        return kwargs
+
+    @pytest.fixture(autouse=True)
+    def setup(self, fxt_output_root, fxt_data_root, fxt_dataset_parameters):
+        self.data_dir = fxt_data_root
+        # self.output_dir = fxt_output_root
+        self.output_dir = Path("/home/gzalessk/code/openvino_xai/tests/perf/validation_results/5000_subset_42_seed")
```

Comment:
is it intentional?

Reply:
Fixed in [c3ed6c9](https://github.com/openvinotoolkit/openvino_xai/pull/73/commits/c3ed6c9d8a358ca9196d5ef94c6e9cffe3f2d994)